### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by uisendy**
### Leaderboard: setup project
**In this activity JavaScript project for the Leaderboard list app was setup  using webpack and ES6 features, notably modules. **

The following was achieved. 

- Main structure of the Javascript Leaderboard
- No Linters 
- Gitflow practises
- Webpack practises
- ES6 Best practises

### CODE REVIEW

File: src/components/AddScore.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <form>
+      <div class="name-input-area relative z-0 mb-6 w-full group">
+        <input
+          type="text"
+          name="name"
+          id="name"
+          class="name-input block py-2.5 px-0 w-full text-sm text-gray-900 bg-transparent border-0 border-b-2 border-gray-300 appearance-none dark:text-white dark:border-gray-600 dark:focus:border-yellow-400 focus:outline-none focus:ring-0 focus:border-yellow-400 peer"
+          placeholder=" "
+          required=""
+        />
```
Comment (roshan-bajgain) [inline src/components/AddScore.js]: - [ ] According to the project requirement, the HTML code should look like this in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), Please make the refresh buttons and Recent scores. See the image below for reference.
### Buttons
fimga

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @uisendy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddScore.js (hunk 1 above)
Comment (uisendy) [reply src/components/AddScore.js]: Ohhh. i missed that. Thanks so much. I appreciate

**Review by phelian23 — APPROVED** [review]
Hi @uisendy,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**